src/components/MethodTable.js

```diff
@@ -42,6 +44,10 @@ const ContentEl = styled.div`
   padding: 0;
   padding-top: 1rem;
   padding-bottom: 1.5rem;
+
+  ${Code}, ${Table} {
+    padding-left: 1.5rem;
```

Comment:
I shifted this down to fine-tune rendering. Without word wrap, this whole thing will scroll, and by putting the padding on the inner element, we avoid having 1.5rem of dead space on the left side of the code:
![Screen Shot 2020-02-25 at 11 15 29 AM](https://user-images.githubusercontent.com/1551487/75266307-28ea4300-57c0-11ea-9b50-351d57df84a2.png)